src/client/components/multi-form-editor/index.js

```diff
@@ -1,25 +1,113 @@
 
 const React = require('react');
 const h = require('react-hyperscript');
-const MultiStep = require('react-stepzilla').default;
+const { Tab, Tabs, TabList, TabPanel } = require('react-tabs');
+const { Link } = require('react-router-dom');
 
-const Editor = require('./editor');
+
+const Editor = require('../editor');
 const SubmitPage = require('./submit-page');
 
-const steps = [
-  {name: 'Edit', component: h(Editor)},
-  {name: 'Submit', component: h(SubmitPage)}
-];
+class Context extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      interactions: []
+    };
+  }
+
+  addInteractionForm() {
+    this.setState({
+      interactions: [...this.state.interactions, {
+        entity0: '?',
+        interaction: 'binds',
+        entity1: '?'
+      }]
+    });
+  }
 
-class MultiFormEditor extends React.Component {
   render() {
-    return h('div', [
-      h(MultiStep, {
-        steps: steps,
-        showSteps: false
-      })
-    ]);
+    const interactions = this.state.interactions.map(interaction => h('div.interaction', [
+      h('div', interaction.entity0),
+      h('div', interaction.interaction),
+      h('div', interaction.entity1)
+    ]));
+    return (
+      h('div.editor-context', [
+        h('div.site-logo', [
+          h('i.icon.icon-logo.home-logo-icon'),
+          ' Factoid'
+        ]),
+        h('div.interactions', [
+          ...interactions,
+          h('button', {onClick: e => this.addInteractionForm()}, 'New Form Entry')
+        ]),
+      ])
+    );
   }
 }
 
-module.exports = MultiFormEditor;
+
+class MFE extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      doc: null,
```

Comment:
Is this part of `stylelint-config-standard`? Or another standard? (specifically I mean instantiating to `null` instead of `undefined`)